sdk/resourcemanager/azure-resourcemanager-authorization/src/main/java/com/azure/resourcemanager/authorization/models/RoleAssignmentProperties.java

```diff
@@ -25,6 +25,9 @@ public final class RoleAssignmentProperties {
     @JsonProperty(value = "principalId", required = true)
     private String principalId;
 
+    @JsonProperty(value = "description")
+    private String description;
```

Comment:
This is through code regen?

Reply:
No. Currently this part is kind of hybrid of 2015-07-01 and 2020-10-01-preview (based on 2015-07-01, but add "condition" and switched api-version in API call to 2020-10-01-preview).

We probably going to regen it with 2020-10-01-preview, when requested for 2022-04-01 ("description" in 2022-04-01)

Reply:
Seems they already have "condition" in their stable swagger?
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/authorization/resource-manager/Microsoft.Authorization/stable/2022-04-01/authorization-RoleAssignmentsCalls.json#L594-L597

Reply:
OK, that would be great.

I thought I've tried the codegen with 2022-04-01...
But we will still wait for the request for a while, from .NET they've got report that some region was not available in 2022-04-01

Reply:
ok